plugins/council/skills/convene/tests/run-tests.sh

```
@@ -0,0 +1,284 @@
+
+# GNU-only date math would break on macOS; epoch seconds behave the same on both.
+# Only the positive intent is asserted here — check-shell-portability.mjs already
+# owns "no unguarded GNU-only construct", and it distinguishes a real invocation
+# from prose that merely names one. Re-implementing that distinction with grep
+# would just false-flag this file's own explanation of why epochs are used.
+has "TTL uses epoch arithmetic"  'checked_at_epoch'
+
+# Nothing set in one bash block survives into the next; the seat blocks must say so.
+has "seat blocks re-hydrate their pins"  'DIR=$(cat ".claude/state/council-run-$RUN_KEY")'
+# Two concurrent councils in one repo must not share a run pointer.
+has "run pointer is keyed by session"    'DIR=$(cat ".claude/state/council-run-$RUN_KEY")'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)